pages/common/logstash.md

```diff
@@ -0,0 +1,21 @@
+# logstash
+
+> An ETL tool, usually used to ingest data from various sources, like databases and flat log files into elasticsearch.
+
+- Boilerplate configuration for logstash to provide logic for extract, transform and load, respectively:
```

Comment:
We usually give command descriptions along with their examples here. This seems to be the configuration for logstash. Since, you are already mentioning the same thing in the last example, I think this is not needed anymore. 

Also, removing this would solve #1263